benchmarker/modules/do_pytorch.py

```diff
@@ -77,28 +75,31 @@ def run_internal(self):
         device = torch.device("cuda" if self.params["gpus"] else "cpu")
 
         x_train, y_train = self.load_data()
-        # TODO: make of/on-core optional
-        self.x_train = torch.from_numpy(x_train).to(device)
-        self.y_train = torch.from_numpy(y_train).to(device)
+
         # train_dataset = torch.utils.data.TensorDataset(x_train, y_train)
         # train_loader = torch.utils.data.DataLoader(train_dataset, batch_size=self.params["batch_size"], shuffle=False)
 
         model = self.net
         if len(self.params["gpus"]) > 1:
             model = nn.DataParallel(model)
+        # TODO: make of/on-core optional
+        self.x_train = torch.from_numpy(x_train).to(device)
+        self.y_train = torch.from_numpy(y_train).to(device)
```

Comment:
Could we copy `y_train` to device only if we do training... it is not needed for evaluation?!